provider/resources.go

```diff
@@ -232,6 +233,9 @@ func Provider() tfbridge.ProviderInfo {
 		},
 	}
 
+	err := prov.ComputeDefaults(tfbridge.TokensSingleModule(
+		"newrelic_", mainMod, tfbridge.MakeStandardToken(mainPkg)))
+	contract.AssertNoError(err)
```

Comment:
This will put all tokens in a single module. Looking at the provider, it seems like we want to put different tokens into different modules. I would look into `tfbridge.TokensKnownModules`.